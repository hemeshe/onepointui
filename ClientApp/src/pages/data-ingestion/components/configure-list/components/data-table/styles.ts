import styled from "styled-components/macro";

import * as T from "../../../../../../components/table";

export const SortButton = styled.span`
  cursor: pointer;
  margin-left: 10px;
`;

// export const Input = styled.textarea`
//   background: #e6e6e6;
//   border: 1px solid #c2c2c2;
//   padding: 5px;
//   border-radius: 4px;
//   display: block;
//   width: 100%;
//   height: auto;
// `;

export const TD = styled(T.TD)``;

export const RemoveAddInput = styled.span`
  cursor: pointer;
`;

export const Select = styled.select`
  background: #e6e6e6;
  border: 1px solid #c2c2c2;
  padding: 5px;
  border-radius: 4px;
  display: block;
`;

export const Option = styled.option``;
