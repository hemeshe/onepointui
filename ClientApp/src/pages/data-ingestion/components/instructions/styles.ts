import styled, { css } from 'styled-components';

export const Container = styled.section`
  display: flex;
  flex-direction: column;
`;

export const Section = styled.section`
  display: flex;
  flex-direction: column;
  margin: 0 0 20px 0;
`;

export const List = styled.ul<{ listStyle?: string }>`
  color: #7f7f7f;
  margin: 0;
  font-size: ${({ theme }) => theme.fontSize['m']};
  padding: 0;

  ${({ listStyle }) => {
    if (listStyle) {
      return css`
        list-style: ${listStyle};
      `;
    }
  }};
`;

export const ListItem = styled.li`
  list-styles-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'%3e%3cpath fill='none' stroke='%23343a40' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M2 5l6 6 6-6'/%3e%3c/svg%3e");
  padding: 5px;
  color: #4a4949;

  :before {
    content: '-';
    padding-right: 4px;
  }
`;

export const SmallTitle = styled.span`
  color: #a50e0e;
  font-size: ${({ theme }) => theme.fontSize['m']};
  font-weight: 800;
  display: block;
  width: 100%;
  margin: 0 0 10px 0;
`;

export const Para = styled.span`
  color: #7f7f7f;
  font-size: ${({ theme }) => theme.fontSize['m']};
  padding-top: 4px;
`;

export const ReportContainer = styled.div`
  display: flex;
  padding: 0 0 0 10px;
`;

export const ReportTitle = styled(SmallTitle)`
  font-weight: 400;
  font-size: ${({ theme }) => theme.fontSize['l']} !important;
`;

export const ReportImageContainer = styled.div``;

export const ReportImage = styled.img`
  max-width: 100%;
  height: auto;
`;

export const ClickHere = styled.span`
  cursor: pointer;
  font-family: Open Sans;
  font-weight: bold;
  font-size: font-size: ${({ theme }) => theme.fontSize['m']};
  color: #0F488F;
`;
