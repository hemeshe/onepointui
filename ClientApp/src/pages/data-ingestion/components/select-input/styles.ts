import styled from "styled-components";

export const StyledInput = styled.div`
  display: flex;
  flex-direction: column;
  position: relative;
`;

export const Label = styled.span`
  display: block;
  width: 100%;
  padding: 10px 2px 2px 0px;
`;
