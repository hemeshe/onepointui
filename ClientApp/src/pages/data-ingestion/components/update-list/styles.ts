import styled from "styled-components";

export const Container = styled.div`
  display: flex;
  flex-direction: column;
  width: 100%;
  margin-left: 20px;
  margin-right: 20px;
  overflow: auto;
`;

export const Row = styled.div`
  flex: 0 0 auto;
  width: 100%;
  padding: 10px;

  @media (min-width: 576px) {
    width: 50%;
  }

  @media (min-width: 768px) {
    width: 50%;
  }

  @media (min-width: 992px) {
    width: 50%;
  }

  @media (min-width: 1200px) {
    width: 50%;
  }

  @media (min-width: 1400px) {
    width: 50%;
  }
`;

export const RowContent = styled.div`
  padding: 20px 100px;
`;

export const Bottom = styled.div<{ justifyContent?: string }>`
  display: flex;
  justify-content: ${({ justifyContent }) =>
    justifyContent ? justifyContent : "flex-end"};
  margin-top: 50px;
`;

export const Button = styled.a`
  outline: none;
  border: none;
  background: #a6a6a6;
  border-radius: 4px;
  color: white;
  letter-spacing: 1px;
  padding: 10px 30px;
  font-size: ${({ theme }) => theme.fontSize["xs"]};
  font-weight: bold;
  cursor: pointer;
  text-decoration: none;
`;
