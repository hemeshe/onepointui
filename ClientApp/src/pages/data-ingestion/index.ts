export { DataIngestion } from "./data-ingestion";
