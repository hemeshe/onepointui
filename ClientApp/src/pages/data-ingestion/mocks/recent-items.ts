export const recentItems = [
  {
    historyId: 727,
    fileName: "process_599af217-6c59-4680-a661-daf602584ade.csv",
    originFrom: "DataIngestion",
    changeTime: "2021-06-02T11:50:32",
    fileSize: "1.7KB",
    fileType: "Trade Book to Financials",
    classOfBusiness: "Oil",
    submittedBy: "Parameswar.Naik@shell.com",
    fileStatus: "Uploaded",
    reasonForFailure: null,
  },
  {
    historyId: 860,
    fileName: "process_48db6572-8ebf-4ddf-a367-585aa49aee0e.csv",
    originFrom: "DataIngestion",
    changeTime: "2021-06-02T11:44:12",
    fileSize: "1.5KB",
    fileType: "Risk Margin Weekly",
    classOfBusiness: "Oil",
    submittedBy: "Parameswar.Naik@shell.com",
    fileStatus: "Uploaded",
    reasonForFailure: null,
  },
  {
    historyId: 745,
    fileName: "process_724b04ae-e66c-43ff-b5ca-2a1ba5d777b3.csv",
    originFrom: "DataIngestion",
    changeTime: "2021-05-28T06:46:53",
    fileSize: "103B",
    fileType: "Risk Margin Weekly",
    classOfBusiness: "Oil",
    submittedBy: "Muthukumar.KM@shell.com",
    fileStatus: "Processed",
    reasonForFailure: null,
  },
  {
    historyId: 904,
    fileName: "process_10c64eb5-2f58-4f63-9e95-f8a9919c4555.csv",
    originFrom: "DataIngestion",
    changeTime: "2021-05-27T13:19:09",
    fileSize: "103B",
    fileType: "Risk Margin Weekly",
    classOfBusiness: "Oil",
    submittedBy: "Muthukumar.KM@shell.com",
    fileStatus: "Processed",
    reasonForFailure: null,
  },
  {
    historyId: 469,
    fileName: "process_afb9f5ae-9688-47c3-b7d4-079afffdc205.csv",
    originFrom: "DataIngestion",
    changeTime: "2021-05-27T13:18:23",
    fileSize: "220B",
    fileType: "Other Contributions Plan(Before Tax)",
    classOfBusiness: "Oil",
    submittedBy: "Muthukumar.KM@shell.com",
    fileStatus: "Processed",
    reasonForFailure: null,
  },
];
