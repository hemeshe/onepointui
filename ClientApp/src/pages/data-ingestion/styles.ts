import styled from "styled-components";
import { Card } from "../../components/card";

export const Main = styled(Card)`
  margin: 0px auto;
  display: flex;
  flex-direction: column;
  width: 99%;
`;
export const Top = styled.div`
  display: flex;
  justify-content: space-between;

  margin-left: 20px;
  margin-right: 20px;
`;

export const TopLeft = styled.div``;
export const TopRight = styled.div`
  display: flex;
  align-items: center;
  justify-content: space-around;
`;

export const Top2 = styled.div`
  display: flex;
  justify-content: space-between;
`;

export const CardNavRight = styled.div`
  padding-right: 5px;
`;

export const CardNavLeft = styled.div``;

export const Button = styled.button<{ backgroundColor?: string }>`
  outline: none;
  border: none;
  background: ${({ backgroundColor }) =>
    backgroundColor ? backgroundColor : "#e6e6e6"};
  border-radius: 4px;
  color: black;
  letter-spacing: 1px;
  padding: 6px 12px;
  font-size: ${({ theme }) => theme.fontSize["xs"]};
  cursor: pointer;
  margin: 2px 8px;
`;
