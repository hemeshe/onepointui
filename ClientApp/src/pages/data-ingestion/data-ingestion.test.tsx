import React from "react";
import { screen, waitFor } from "@testing-library/react";

import { DataIngestion } from "./data-ingestion";
import { renderWithTheme } from "../../styles/render-with-theme";

import { recentItems } from "./mocks/recent-items";

describe("<DataIngestion />", () => {
  it("should render correctly", () => {
    const { container } = renderWithTheme(<DataIngestion />);
    expect(container).toMatchSnapshot();
  });

  it("should render page tags", () => {
    renderWithTheme(<DataIngestion />);
    const pageTitle = screen.getByRole("title");
    const RecentlyAdded = screen.getByText("Recently Added");
    expect(pageTitle).toHaveTextContent("Data Ingestion");
    expect(RecentlyAdded).toBeInTheDocument();
  });

  it("should render recently added items", async () => {
    renderWithTheme(<DataIngestion />);

    //const div = await waitFor(() => screen.getByTestId('recenty-added'));

    // await waitFor(() => {
    //   recentItems.forEach((mock) => {
    //     expect(screen.getByText(mock.fileType, { exact: false })).toBeInTheDocument();
    //   });
    // });

    //  recentItemNames.forEach((el, i) => {
    //   expect(el.innerHTML).toBe(recentItems[i].fileType);
    //  })
    // await waitFor(() => {
    //   mockUsers.forEach((mockUser) => {
    //     expect(screen.getByText(mockUser.name, { exact: false })).toBeInTheDocument();
    //   });
    // });
  });
});
