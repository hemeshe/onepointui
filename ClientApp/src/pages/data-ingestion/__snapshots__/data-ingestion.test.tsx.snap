// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DataIngestion /> should render correctly 1`] = `
.c20 {
  background: #ffffff;
  box-shadow: 0px 4px 10px rgba(0,0,0,0.04),0px 1px 3px rgba(0,0,0,0.08), 0px 0px 1px rgba(0,0,0,0.12);
  border-radius: 3px;
}

.c22 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  box-sizing: border-box;
  background: #ffffff;
  padding: 0 15px 0px 5px;
}

.c32 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  background: #f5f5f5;
}

.c24 {
  color: #404040;
  font-size: 1.25rem;
}

.c27 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  box-sizing: border-box;
}

.c30 {
  height: 100%;
  border-bottom: none;
  font-weight: normal;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  box-sizing: border-box;
  margin: 0px 30px 0 0;
  line-height: 1.25rem;
  padding: 5px;
}

.c28 {
  height: 100%;
  border-bottom: 3px solid #009EB4;
  font-weight: bold;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  box-sizing: border-box;
  margin: 0px 30px 0 0;
  line-height: 1.25rem;
  padding: 5px;
}

.c29 {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #595959;
  font-size: 1.25rem;
  cursor: pointer;
}

.c21 {
  margin: 0px auto;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 99%;
}

.c23 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  margin-left: 20px;
  margin-right: 20px;
}

.c25 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
}

.c26 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c31 {
  padding-right: 5px;
}

.c0 {
  width: 100%;
  padding-right: 1rem;
  padding-left: 1rem;
  margin-right: auto;
  margin-left: auto;
}

.c1 {
  height: 55px;
  width: 100%;
  position: fixed;
  background-color: #FFFFFF;
  top: 0;
  left: 0;
  color: #595959;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  z-index: 1;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  height: 55px;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 20%;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-family: Futura Medium;
  color: #003c88;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin: 0 20px;
  font-size: 1.25rem;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  width: 100%;
  font-weight: normal;
  font-size: 0.875rem;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
}

.c9 {
  padding: 0 4px;
}

.c6 {
  font-weight: 900;
  word-spacing: 0.1rem;
  -webkit-letter-spacing: 0.1rem;
  -moz-letter-spacing: 0.1rem;
  -ms-letter-spacing: 0.1rem;
  letter-spacing: 0.1rem;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 100%;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 60%;
  background: none;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 100%;
}

.c11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  height: 100%;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c12 {
  height: 100%;
  border-bottom: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  box-sizing: border-box;
  margin: 14px 20px;
  font-weight: normal;
}

.c14 {
  height: 100%;
  border-bottom: 3px solid #009EB4;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  box-sizing: border-box;
  margin: 14px 20px;
  font-weight: bold;
}

.c13 {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #003c88;
  font-size: 1rem;
}

.c15 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 20%;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
}

.c16 {
  display: inline-block;
  position: relative;
}

.c17 {
  -webkit-text-decoration: none;
  text-decoration: none;
  margin: 4px 8px;
  color: #003c88;
  position: relative;
}

.c4 {
  height: 40px;
  width: 40px;
  fill: currentColor;
}

.c18 {
  height: 18px;
  width: 18px;
  fill: currentColor;
}

.c19 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  position: relative;
}

.c40 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1 1 auto;
  -ms-flex: 1 1 auto;
  flex: 1 1 auto;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  --bs-gutter-x: 1.5rem;
  --bs-gutter-y: 0;
  margin-top: calc(var(--bs-gutter-y) * -1);
  margin-right: calc(var(--bs-gutter-x) / -2);
  margin-left: calc(var(--bs-gutter-x) / -2);
  -webkit-box-pack: stretch;
  -webkit-justify-content: stretch;
  -ms-flex-pack: stretch;
  justify-content: stretch;
}

.c40 > * {
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  width: 100%;
  max-width: 100%;
  padding-right: calc(var(--bs-gutter-x) / 2);
  padding-left: calc(var(--bs-gutter-x) / 2);
  margin-top: var(--bs-gutter-y);
}

.c41 {
  -webkit-flex: 1 0 0%;
  -ms-flex: 1 0 0%;
  flex: 1 0 0%;
  position: relative;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
}

.c33 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  margin-left: 20px;
  margin-right: 20px;
  overflow: auto;
  margin-bottom: 20px;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  max-height: calc(100vh - 170px);
}

.c34 {
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  width: 100%;
  padding: 10px;
  box-sizing: border-box;
}

.c35 {
  padding: 20px 60px;
  box-sizing: border-box;
}

.c45 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  margin-top: 50px;
}

.c46 {
  outline: none;
  border: none;
  background: #a6a6a6;
  border-radius: 4px;
  color: white;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  padding: 10px 30px;
  font-size: 0.75rem;
  font-weight: bold;
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c36 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  position: relative;
}

.c37 {
  display: block;
  width: 100%;
  padding: 10px 2px 2px 0px;
}

.c38 {
  display: block;
  width: 100%;
  height: calc(1.5em + 0.75rem + 2px);
  padding: 0.375rem 1.75rem 0.375rem 0.75rem;
  font-size: 1rem;
  font-weight: 400;
  line-height: 1.5;
  color: #495057;
  vertical-align: middle;
  background-color: #fff;
  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'%3e%3cpath fill='none' stroke='%23343a40' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M2 5l6 6 6-6'/%3e%3c/svg%3e");
  background-repeat: no-repeat;
  background-position: right 0.75rem center;
  background-size: 16px 12px;
  border: 1px solid #ced4da;
  border-radius: 2rem;
  -webkit-appearance: none;
  -moz-appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  outline: 0;
}

.c38::-ms-expand {
  display: none;
}

.c39 {
  width: 100%;
  padding: 5px;
  color: #595959;
  display: block;
}

.c42 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  position: relative;
  overflow: scroll;
}

.c44 {
  width: 100%;
  padding: 5px;
  background: #e6e6e6;
  border-radius: 5px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  min-height: 125px;
  height: auto;
  border: 2px solid #e6e6e6;
  overflow: auto;
}

.c43 {
  display: block;
  width: 100%;
  padding: 10px 2px;
}

.c47 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c48 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  margin-bottom: 20px;
}

.c49 {
  color: #404040;
  font-size: 1rem;
  font-weight: 800;
  display: block;
  width: 100%;
}

@media (max-width:1280px) {
  .c4 {
    max-height: 56px;
  }
}

@media (max-width:1280px) {
  .c18 {
    max-height: 56px;
  }
}

@media (min-width:576px) {
  .c41 {
    -webkit-flex: 0 0 auto;
    -ms-flex: 0 0 auto;
    flex: 0 0 auto;
    -webkit-flex: 0 0 auto;
    -ms-flex: 0 0 auto;
    flex: 0 0 auto;
    width: 50%;
  }
}

@media (min-width:768px) {
  .c41 {
    -webkit-flex: 0 0 auto;
    -ms-flex: 0 0 auto;
    flex: 0 0 auto;
    -webkit-flex: 0 0 auto;
    -ms-flex: 0 0 auto;
    flex: 0 0 auto;
    width: 50%;
  }
}

@media (min-width:992px) {
  .c41 {
    -webkit-flex: 0 0 auto;
    -ms-flex: 0 0 auto;
    flex: 0 0 auto;
    -webkit-flex: 0 0 auto;
    -ms-flex: 0 0 auto;
    flex: 0 0 auto;
    width: 50%;
  }
}

@media (min-width:1200px) {
  .c41 {
    -webkit-flex: 0 0 auto;
    -ms-flex: 0 0 auto;
    flex: 0 0 auto;
    -webkit-flex: 0 0 auto;
    -ms-flex: 0 0 auto;
    flex: 0 0 auto;
    width: 50%;
  }
}

@media (min-width:1400px) {
  .c41 {
    -webkit-flex: 0 0 auto;
    -ms-flex: 0 0 auto;
    flex: 0 0 auto;
    -webkit-flex: 0 0 auto;
    -ms-flex: 0 0 auto;
    flex: 0 0 auto;
    width: 50%;
  }
}

@media (min-width:576px) {
  .c34 {
    width: 50%;
  }
}

@media (min-width:768px) {
  .c34 {
    width: 50%;
  }
}

@media (min-width:992px) {
  .c34 {
    width: 50%;
  }
}

@media (min-width:1200px) {
  .c34 {
    width: 50%;
  }
}

@media (min-width:1400px) {
  .c34 {
    width: 50%;
  }
}

<div>
  <section
    class="c0"
  >
    <header
      class="c1"
    >
      <section
        class="c0"
      >
         
        <section
          class="c2"
        >
          <div
            class="c3"
          >
            <div
              class=""
            >
              <svg
                class="c4"
                size="44"
              >
                logo.svg
              </svg>
            </div>
            <div
              class="c5"
            >
              <div
                class="c6"
                data-testid="title"
              >
                T&S MI PORTAL
              </div>
              <div
                class="c7"
              >
                <div
                  class="c8"
                >
                  By : 
                </div>
                <div
                  class="c9"
                >
                   T&S FINANCE
                </div>
              </div>
            </div>
          </div>
          <div
            class="c10"
          >
            <div
              class="c11"
            >
              <span
                class="c12"
              >
                <a
                  class="c13"
                  href="/home"
                >
                  Home
                </a>
              </span>
              <span
                class="c12"
              >
                <a
                  class="c13"
                  href="/commentary"
                >
                  Commentary
                </a>
              </span>
              <span
                class="c12"
              >
                <a
                  class="c13"
                  href="/mapping"
                >
                  Mapping
                </a>
              </span>
              <span
                class="c14"
              >
                <a
                  class="c13"
                  href="/data-ingestion"
                >
                  Data Ingestion
                </a>
              </span>
            </div>
          </div>
          <div
            class="c15"
          >
            <div
              class="c16"
            >
              <a
                class="c17"
                href="/"
              >
                <svg
                  class="c18"
                  size="18"
                >
                  info.svg
                </svg>
              </a>
              <div
                class="c19"
              />
            </div>
          </div>
        </section>
      </section>
    </header>
    <section
      class="c20 c21"
    >
      <div
        class="c22"
      >
        <div
          class="c23"
        >
          <div
            class=""
          >
            <h3
              class="c24"
              role="title"
            >
              Data Ingestion 
              
               
              
               
              
               
              
            </h3>
          </div>
          <div
            class="c25"
          />
        </div>
        <div
          class="c26"
        >
          <div
            class=""
          >
            <div
              class="c27"
              style="margin-left: 20px; margin-right: 20px;"
            >
              <span
                class="c28"
              >
                <a
                  class="c29"
                >
                  View
                </a>
              </span>
              <span
                class="c30"
              >
                <a
                  class="c29"
                >
                  Add
                </a>
              </span>
              <span
                class="c30"
              >
                <a
                  class="c29"
                >
                  Edit
                </a>
              </span>
              <span
                class="c30"
              >
                <a
                  class="c29"
                >
                  Configure
                </a>
              </span>
              <span
                class="c30"
              >
                <a
                  class="c29"
                >
                  History
                </a>
              </span>
            </div>
          </div>
          <div
            class="c31"
          />
        </div>
      </div>
      <div
        class="c32"
      >
        <div
          class="c33"
        >
          <div
            class="c34"
            style="border-right: 1px solid #595959;"
          >
            <div
              class="c35"
            >
              <div
                class="c36"
              >
                <span
                  class="c37"
                >
                  Class Of Business
                </span>
                <select
                  class="c38"
                >
                  <option
                    class="c39"
                    value="0"
                  >
                    Loading...
                  </option>
                </select>
              </div>
              <div
                class="c36"
              >
                <span
                  class="c37"
                >
                  Data Ingestion File
                </span>
                <select
                  class="c38"
                >
                  <option
                    class="c39"
                    value="0"
                  >
                    Select class of business first
                  </option>
                </select>
              </div>
              <section
                class="c40"
              >
                <section
                  class="c41"
                  size="6"
                >
                  <div
                    class="c36"
                  >
                    <span
                      class="c37"
                    >
                      Year
                    </span>
                    <select
                      class="c38"
                    >
                      <option
                        class="c39"
                        value="0"
                      >
                        Select data ingestion file first
                      </option>
                    </select>
                  </div>
                </section>
                <section
                  class="c41"
                  size="6"
                  style="padding-left: 5px;"
                >
                  <div
                    class="c36"
                  >
                    <span
                      class="c37"
                    >
                      Month
                    </span>
                    <select
                      class="c38"
                    >
                      <option
                        class="c39"
                        value="Select month"
                      >
                        Select month
                      </option>
                      <option
                        class="c39"
                        value="Jan"
                      >
                        January
                      </option>
                      <option
                        class="c39"
                        value="Feb"
                      >
                        February
                      </option>
                      <option
                        class="c39"
                        value="Mar"
                      >
                        March
                      </option>
                      <option
                        class="c39"
                        value="Apr"
                      >
                        April
                      </option>
                      <option
                        class="c39"
                        value="May"
                      >
                        May
                      </option>
                      <option
                        class="c39"
                        value="Jun"
                      >
                        June
                      </option>
                      <option
                        class="c39"
                        value="Jul"
                      >
                        July
                      </option>
                      <option
                        class="c39"
                        value="Aug"
                      >
                        August
                      </option>
                      <option
                        class="c39"
                        value="Sep"
                      >
                        September
                      </option>
                      <option
                        class="c39"
                        value="Oct"
                      >
                        October
                      </option>
                      <option
                        class="c39"
                        value="Nov"
                      >
                        November
                      </option>
                      <option
                        class="c39"
                        value="Dec"
                      >
                        December
                      </option>
                    </select>
                  </div>
                </section>
              </section>
              <div
                class="c42"
              >
                <span
                  class="c43"
                >
                  Preview of selected Ingestion Table
                </span>
                <div
                  class="c44"
                />
              </div>
              <div
                class="c45"
              >
                <a
                  class="c46"
                  href="/data-ingestion/view-list/////undefined/undefined"
                >
                  VIEW
                </a>
              </div>
            </div>
          </div>
          <div
            class="c34"
          >
            <div
              class="c35"
            >
              <section
                class="c47"
              >
                <div
                  class="c48"
                >
                  <span
                    class="c49"
                  >
                    Recently Added
                  </span>
                </div>
              </section>
            </div>
          </div>
        </div>
      </div>
    </section>
  </section>
</div>
`;
